src/array.jl

```diff
@@ -854,7 +841,15 @@ end
 Drop levels which do not appear in categorical array `A` (so that they will no longer be
 returned by [`levels`](@ref DataAPI.levels)).
 """
-droplevels!(A::CategoricalArray) = levels!(A, intersect(levels(A), uniquelevels(A)))
+function droplevels!(A::CategoricalArray)
+    # FIXME temporary code until PR#359 is merged
```

Comment:
Just a note so that we don't forget this before merging (x-ref #359).

Reply:
I've just rebased the PR against the updated `droplevels!()` and squashed the commits, so it should be ready for merging (when it's time for a new minor release).